backend/main/leaderboard_test.go

```diff
@@ -34,4 +32,71 @@ func TestGetCommunityLeaderboard(t *testing.T) {
 	assert.Equal(t, 2, len(p.Data))
 	assert.Equal(t, 3, p.Data[0].Score)
 	assert.Equal(t, 2, p.Data[1].Score)
-}
\ No newline at end of file
+}
+
+func TestGetCommunityLeaderboardWithEarlyVotes(t *testing.T) {
+	clearTable("communities")
+	clearTable("community_users")
+	clearTable("community_users_achievements")
+	clearTable("proposals")
+	clearTable("votes")
+
+	communityId := otu.AddCommunities(1)[0]
+	earlyVoteBonus := 1
+
+	otu.GenerateLeaderboardWithEarlyVotes(communityId)
+
+	response := otu.GetCommunityLeaderboardAPI(communityId)
+	checkResponseCode(t, http.StatusOK, response.Code)
+
+	var p test_utils.PaginatedResponseWithLeaderboardUser
+	json.Unmarshal(response.Body.Bytes(), &p)
+
+	assert.Equal(t, 2, len(p.Data))
+	assert.Equal(t, 2+2*earlyVoteBonus, p.Data[0].Score)
+	assert.Equal(t, 2, p.Data[1].Score)
+}
+
+func TestGetCommunityLeaderboardWithSingleStreak(t *testing.T) {
+	clearTable("communities")
+	clearTable("community_users")
+	clearTable("community_users_achievements")
```

Comment:
just a small thing, but maybe we can call this table `achievements`? to avoid the long name?

Reply:
Done.